npm-shrinkwrap.json

```diff
@@ -5738,6 +5672,15 @@
         }
       }
     },
+    "cpu-features": {
```

Comment:
I think that this newly added `cpu-features` dependency is the cause of the following test error:
```
make: Entering directory '/tmp/build/c694c2a4/versioned-source/node_modules/cpu-features/build'
  ACTION Configuring dependencies /tmp/build/c694c2a4/versioned-source/node_modules/cpu-features/deps/cpu_features/build/Makefile
/bin/sh: 1: cmake: Permission denied
config_deps.target.mk:13: recipe for target '/tmp/build/c694c2a4/versioned-source/node_modules/cpu-features/deps/cpu_features/build/Makefile' failed
make: *** [/tmp/build/c694c2a4/versioned-source/node_modules/cpu-features/deps/cpu_features/build/Makefile] Error 127
make: Leaving directory '/tmp/build/c694c2a4/versioned-source/node_modules/cpu-features/build'
```
`cpu-features` is an ***optional*** dependency (`"optional": true,`) of the pre-existing `ssh2` dependency, so perhaps this `cpu-features` block could be simply deleted without harmful side effects?